src/plugin.py

```diff
@@ -144,9 +154,58 @@ async def get_owned_games(self):
             return [g.in_galaxy_format() for g in self._owned_games.values()]
 
     async def get_subscriptions(self):
-        return [
-            Subscription(SUBSCRIPTIONS.TROVE, subscription_discovery=SubscriptionDiscovery.USER_ENABLED)
-        ]
+        subscriptions: List[Subscription] = []
+        current_month_unlocked = False
+        current_or_former_subscriber = await self._api.had_subscription()
+
+        if current_or_former_subscriber:
+            async for product in await self._api.get_subscription_products_with_gamekeys():
+                if not product.get('isChoiceTier'):
+                    # no more choice content, now humble montly data coming from before 2020
+                    break
+                subscriptions.append(
+                    Subscription(product['title'], owned=True)
+                )
+                if product.is_active_content:
+                    # assuming only current month has "is_active_content": true
+                    current_month_unlocked = True
+
+        if not current_month_unlocked:
+            '''
+            - for not subscribers as potential discovery of current choice games
+            - for subscribers who has not used "Early Unlock" yet:
+              https://support.humblebundle.com/hc/en-us/articles/217300487-Humble-Choice-Early-Unlock-Games
+            '''
+            choice_months_details = await self._api.get_choice_month_details()
+            active_month = choice_months_details['active_month']
+
+            if current_or_former_subscriber:
+                active_month_content = await self._api.get_choice_content_data(
+                    active_month['montly_product_page_url']
+                )
+                if active_month_content.user_subscription_plan is not None:
+                    current_month_unlocked = True
+                end_time = None  # owned subscription never ends
+            else:
+                # TODO the nearest first Friday of month at 10 am PT
+                end_time = None  # tell new commers to hurry up
+
+            subscriptions.append(
+                Subscription(
+                    active_month['short_human_name'],
+                    owned=current_month_unlocked,
+                    end_time=end_time
```

Comment:
wrong. True only when NOT `current_month_unlocked` AND NOT `is_going_to_be_unlocked`